test_e2e.py

```diff
@@ -12,13 +12,18 @@
 
 @pytest.fixture(scope="module")
 def driver():
- with webdriver.Firefox() as driver:
-     yield driver
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    opts.add_argument('--disable-dev-shm-usage')
+    with webdriver.Chrome('./chromedriver', options=opts) as driver:
```

Comment:
You can do `webdriver.Chrome(options=opts)` so that it checks both PATH and the current directory